lib/model/folder.go

```diff
@@ -698,6 +695,22 @@ func (f *folder) basePause() time.Duration {
 	return time.Duration(f.PullerPauseS) * time.Second
 }
 
+// resetPullFailTimer reschedules pull retrying with increasingly longer pauses
+// between tries (with lower limit based on pull duration and fix upper limit).
+func (f *folder) resetPullFailTimer(timer *time.Timer, pause, pullDur time.Duration) time.Duration {
+	max := 60 * f.basePause()
+	// Pause at least as long as the time the previous (failed) pull took
+	for pullDur > pause && pause < max {
+		pause *= 2
```

Comment:
This method is confusing. There are multiple opportunities to return a pause > max and in some cases we multiply the pause by four in one pass... 

I think it would be simpler to keep the easy sequence on `pause` (1, 2, 4 etc minutes up to max) and always sleep `pullDur + pause`?